lib/shared/addon/components/container/form-volumes/component.js

```diff
@@ -37,25 +37,25 @@ export default Component.extend({
 
   init() {
     this._super(...arguments);
-    this.sendAction('registerHook', this.saveVolumes.bind(this), {name: 'saveVolumes', key: '_volumeHooks'});
+    this.sendAction('registerHook', this.saveVolumes.bind(this), { name: 'saveVolumes', key: '_volumeHooks' });
   },
 
   didInitAttrs() {
-    if (!get(this,'expandFn')) {
-      set(this,'expandFn', function(item) {
-          item.toggleProperty('expanded');
+    if (!get(this, 'expandFn')) {
+      set(this, 'expandFn', function (item) {
+        item.toggleProperty('expanded');
       });
     }
 
     const out = [];
     let entry;
-    (get(this, 'workload.volumes')||[]).forEach((volume) => {
-      if ( volume.persistentVolumeClaim ) {
+    (get(this, 'workload.volumes') || []).forEach((volume) => {
+      if (volume.persistentVolumeClaim) {
         entry = {
           mode: EXISTING_PVC,
           volume,
         };
-      } else if ( volume.hostPath ) {
+      } else if (volume.hostPath) {
```

Comment:
Do you have some tooling that's making spacing changes?  This is an an odd combination of adding and removing them througout this file.   Spaces are cheap and the standard in this repo is generally more is better
```
if ( condition ) {
  fn(arg1, arg2)
}
<probably a blank line before more logic>
```

Don't go back and undo it all now, but not touching existing ones in the middle of a large PR is preferable.  If you want to bulk cleanup, make it a separate PR.